LICENSE

```diff
@@ -0,0 +1,13 @@
+Copyright Brightcove Inc
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
```

Comment:
The long version of this license used to be in the repo, but all of our other projects use the short one. I think it makes sense to switch here.